blog/2022-01-19-play-with-capact-via-cli.mdx

```diff
@@ -0,0 +1,334 @@
+<TabItem value="policy">
+
+First, save the Policy to the `/tmp/policy.yaml` file:
+
+```yaml title="/tmp/policy.yaml"
+cat <<EOF > /tmp/policy.yaml
+rules:
+  - interface:
+      path: cap.interface.capactio.examples.say
+    oneOf:
+    - implementationConstraints:
+        attributes:
+          - path: cap.attribute.capactio.examples.be-positive
+            revision: 0.1.0
+EOF
+```
+
+Next, create the Action with the Policy:
+
+```bash
+capact act create --name hello cap.interface.capactio.examples.greet --action-policy-from-file /tmp/policy.yaml
+```
+
+</TabItem>
+</Tabs>
+</div>
+
+2. Wait for the Action to have the `READY_TO_RUN` status:
+
+	```bash
+	capact act get hello
+	```
+3. When the status is `READY_TO_RUN`, run the Action:
+
+	```bash
+	capact act run hello
+	```
+4. Watch the progress:
+
+	```bash
+	capact act watch hello
+	```
+5. **(Optional)** Once the Action is finished, copy the name of the Pod for the `print` workflow step (column `PODNAME`). To read its logs, run:
+
+	```bash
+	kubectl logs {PODNAME} main
+	```
+  Example output:
+
+<div className="tab-container-nested">
+<Tabs
+className="disable-tabs"
+groupId="ref"
+defaultValue="def"
+values={[
+{label: 'Default Execution', value: 'def'},
+{label: 'Policy-Changed Execution', value: 'policy'},
+]}>
+<TabItem value="def">
+
+```bash
+ ____________________________
+< message: Hello from Capact >
+ ----------------------------
+    \
+     \
+      \
+                    ##        .
+              ## ## ##       ==
+           ## ## ## ##      ===
+       /""""""""""""""""___/ ===
+  ~~~ {~~ ~~~~ ~~~ ~~~~ ~~ ~ /  ===- ~~~
+       \______ o          __/
+        \    \        __/
+          \____\______/
+```
+
+</TabItem>
+<TabItem value="policy">
+
+```bash
+ __________________________________
+< message: Never gonna give you up >
+ ----------------------------------
+    \
+     \
+      \
+                    ##        .
+              ## ## ##       ==
+           ## ## ## ##      ===
+       /""""""""""""""""___/ ===
+  ~~~ {~~ ~~~~ ~~~ ~~~~ ~~ ~ /  ===- ~~~
+       \______ o          __/
+        \    \        __/
+          \____\______/
+```
+
+</TabItem>
+</Tabs>
+</div>
+
+6. Get the Action output (TypeInstance). Copy the `id` field value:
+
+	```bash
+	capact act get hello -ojson | jq '.Actions[0].output.typeInstances'
+	```
+
+	Example output:
+	```bash {3}
+  [
+    {
+      "id": "08fcaa07-7846-47af-b6a7-7c3818c69656",
+      "typeRef": {
+        "path": "cap.type.capactio.examples.message",
+        "revision": "0.1.0"
+      }
+    }
+  ]
+	```
+
+7. Get the TypeInstance `value.message` field value:
+
+   ```bash
+   capact ti get {ID} -ojson | jq -r '.[0].latestResourceVersion.spec.value.message'
+   ```
+
+8. Delete the Action:
+
+   ```bash
+   capact act delete hello
+   ```
+
+Let's consider another example, as life isn’t all sunshine and rainbows. Sometimes we need a more positively-colored message. Capact allows you to do it with ease ✨
```

Comment:
It's just my personal opinion, but I would change that to something like:

```suggestion
What if you wanted to output a different message, that reminds you about a masterpiece song you can listen to endlessly? With Capact that's really easy ✨
```

(I'm not convinced to that as well, maybe we can find something else) 😄